src/CssToInlineStyles.php

```diff
@@ -33,19 +34,24 @@ public function __construct()
      */
     public function convert($html, $css = null)
     {
-        $document = $this->createDomDocumentFromHtml($html);
+        $document  = $this->createDomDocumentFromHtml($html);
         $processor = new Processor();
 
         // get all styles from the style-tags
-        $rules = $processor->getRules(
-            $processor->getCssFromStyleTags($html)
-        );
+        $styleTagCss = $processor->getCssFromStyleTags($html);
+
+        // proccess style tag css
+        $mediaQueries = $processor->getMediaQueries($styleTagCss);
+        $rules        = $processor->getRules($styleTagCss);
 
-        if ($css !== null) {
-            $rules = $processor->getRules($css, $rules);
+        // process extra css included as string argument to this function
+        if ( $css !== null ) {
+            $mediaQueries = $processor->getMediaQueries($css, $mediaQueries);
+            $rules        = $processor->getRules($css, $rules);
```

Comment:
this line should be reverted. `=` signs have no reason to be aligned.

Reply:
I'll revert this